src/robot/utils/__init__.py

```diff
@@ -60,7 +60,13 @@
                         get_time, get_timestamp, secs_to_timestamp,
                         secs_to_timestr, timestamp_to_secs, timestr_to_secs,
                         parse_time)
-from .robottypes import is_dict_like, is_list_like, is_str_like, type_name
+from .robottypes import (long, bytes, unicode,
+                         UserString, UserList, UserDict,
```

Comment:
Why are long, bytes, unicode, UserString, UserList, UserDict exposed? I didn't see them used anywhere in the code base except for `UserDict` as a base of `NormalizedDict`. `UserDict` can be conditionally imported in `normalizing` and should actually be replaced with `MutableMapping` now that we don't support Python 2.5 anymore.